### ISSUE TITLE
Add support for Mention

### ISSUE THREAD

**Issue by bytrangle** [orlowdev/gatsby-source-notion-api#19]
If I mention a page, for example its title is "A Real World Article", it will be just plain text in the markdownString.

What I think should be rendered:

```
[A Real World Article](/a1b2c3d4)
```
where `a1b2c3d4`.

What do you think?

**Comment by orlowdev** [comment]
Yeah, I thought about it. But the problem is when you mention a page, it doesn't necessarily mean that that the mentioned page is available with current credentials. This can be checked per page, but it would require downloading too much stuff from the Notion API.
